### DISCUSSION TITLE
who play zzz?

### DISCUSSION THREAD

**Discussion by flyhig** [share121/inter-knot#1045] [其他]
How many people are playing zzz n.11

https://github.com/user-attachments/assets/1118ee67-75e3-40b6-9591-988ca0175939

![Screenshot_2024-08-09-20-21-23-994_tv danmaku bili](https://github.com/user-attachments/assets/2de48a0f-a697-4554-b46f-a3ed3456cc52)
u
![Uploading Screenshot_2024-08-09-18-23-33-380_com.tencent.KiHan.jpg…]()

**Comment by VacuolePaoo** [comment]
你为什么不三连！！！

**Comment by fairy-B** [comment]
Oh！ZZZ has received over 44 million appointments worldwide
